### PR TITLE
add information about testing chart certification on a fork

### PR DESCRIPTION
**Pull request by mmulholla**
Updated readme with full instruction for testing on a fork.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -441,13 +441,42 @@ messages.
...
+         name: <chart-name>
+         shortDescription: <chart description>
+         publicPgpKey: null
+      users:
+         - githubUsername: <your github id>
+      vendor:
+         label: <vendor>
+         name: <decriptive name>
+      ```
+   - Note: the pull request will fail and you will need to manually merge it.
```
Comment (tisutisu) [inline docs/README.md:463]: Before this step .. we also need to enable the github actions on the fork. I think for fork creation, it is disabled by default.

**Review by tisutisu — COMMENTED** [review]
Overall LGTM, except that enabling the github actions on fork. Please mention that in a step as well.

File: docs/README.md (hunk 1 above)
Comment (mmulholla) [reply docs/README.md:464]: Added a note for this.

**Comment by mmulholla** [comment]
@tisutisu Think we are ready to merge now.

**Pull request merged**